Setup/Uninstall.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Klaviyo\Reclaim\Setup;
+
+use Magento\Framework\Setup\UninstallInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Filesystem\DirectoryList;
+
+class Uninstall implements UninstallInterface
+{
+    /**
+     * DirectoryList instance
+     * @var \Magento\Framework\Filesystem\DirectoryList $_dir
+     */
+    protected $_dir;
```

Comment:
Is this used? Maybe my own ignorance of PHP but I see `_dir` and `$dir` used but not `$_dir`

Reply:
the $_dir on this line is the same as $this->_dir referenced below; $dir is the name of the variable passed into the constructor